html/changelog.html

```diff
@@ -55,6 +55,16 @@
 		
 		-->
 
+		<div class='commit sansserif'>
+			<h2 class='date'>6 January 2015</h2>
+			<h3 class='author'>Steelpoint updated:</h3>
+			<ul class='changes bgimages16'>
+				<li class='rscadd'>The Emergency Response Team project has been activated by Nanotrasen. The ERT are a seven man squad that can be deployed by the Central Command (admins) to attempt to rescue the station from a overwhelming threat, or to render assistance in dealing with a significant problem.</li>
+				<li class='rscadd'>The ERT consists of a single Commander, two Security Officers, two Engineering Officers and two Medical Officers. The Commander is preselected when the squad is being spawned in, however the remaining six officers are free to select their role in the squad.</li>
+				<li class='rscadd'>Two new Pulse gun variants have been added to the game. They are the Pulse Carbine and the Pulse Pistol, both have significantly smaller ammunition capacities than the Pulse Rifle, however they both can fit in a backpack and the pistol can fit in a pocket. All ERT Officers have a Pulse Pistol for a sidearm, however the Security Officers get a Pulse Carbine as their primary longarm.</li>
+			</ul>
+		</div>
+
```

Comment:
I followed the given instructions but whatever.

How do I revert the changelog then.